cmd/kube-apiserver/app/options/options_test.go

```diff
@@ -97,3 +97,37 @@ func TestAddFlagsFlag(t *testing.T) {
 		t.Errorf("Expected s.EnableSwaggerUI to be true")
 	}
 }
+
```

Comment:
So the test here doesn't really exercise the argument as an input to etcd, this is more a argument test to the api-server.  

Would it be possible to setup the test in the storage/etcd layer to exercise the parameters?